src/bake/DevServer.zig

```diff
@@ -2411,7 +2414,7 @@ pub fn finalizeBundle(
         if (dev.has_tailwind_plugin_hack) |*map| {
             const first_1024 = code.buffer[0..@min(code.buffer.len, 1024)];
             if (std.mem.indexOf(u8, first_1024, "tailwind") != null) {
-                try map.put(dev.allocator, key, {});
+                try map.put(dev.allocator, try dev.allocator.dupe(u8, key), {});
```

Comment:
If dupe succeeds and put fails we leak the key
```suggestion
                const stable_key = try dev.allocator.dupe(u8, key);
                errdefer dev.allocator.free(stable_key);
                try map.put(dev.allocator, stable_key, {});

```

Reply:
also agree on keeping `getOrPut` since this leaks the key if there was an existing entry